### PR TITLE
added line plot for time series

### LINKED ISSUE #57: Show simulation result in a simple plot

### PR DESCRIPTION
**Pull request by soofstad**
Added simple plot for viewing timeseries data

Screenshot_20211028_083623

closes #57

### CODE REVIEW

File: web/custom-plugins/forecast-of-response/src/components/Plots/LinesOverTime.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <ResponsiveContainer width="100%" height="100%">
+        <LineChart data={data}>
+          <CartesianGrid strokeDasharray="2 2" />
+          <XAxis dataKey="timestamp" />
+          <YAxis unit="m" />
+          <Tooltip />
+          <Legend />
+          {/*TODO: Read threshold values from result file*/}
+          <ReferenceLine y={redLine} stroke="red" label="Max" />
+          <ReferenceLine y={orangeLine} stroke="orange" label="Warning" />
```
Comment (KristianKjerstad) [inline web/custom-plugins/forecast-of-response/src/components/Plots/LinesOverTime.tsx:45]: maybe rename redLine -> maxLine and orangeLine -> warningLine or something?

**Comment by KristianKjerstad** [comment]
small comment on UI: maybe make the text in the "Add graph" button appear on a single line?

**Review by KristianKjerstad — APPROVED** [review]
good!

File: web/custom-plugins/forecast-of-response/src/components/Plots/LinesOverTime.tsx (hunk 1 above)
Comment (soofstad) [reply web/custom-plugins/forecast-of-response/src/components/Plots/LinesOverTime.tsx:45]: fixed!

**Pull request merged**